examples/pipeline_wavernn/wavernn.py

```diff
@@ -0,0 +1,586 @@
+    parser.add_argument(
+        "--upsample-scales",
+        default=[5, 5, 11],
+        type=List[int],
+        help="the list of upsample scales",
+    )
+    parser.add_argument(
+        "--n-bits",
+        default=8,
+        type=int,
+        help="the bits of output waveform",
+    )
+    parser.add_argument(
+        "--sample-rate",
+        default=22050,
+        type=int,
+        help="the rate of audio dimensions (samples per second)",
+    )
+    parser.add_argument(
+        "--hop-length",
+        default=275,
+        type=int,
+        help="the number of samples between the starts of consecutive frames",
+    )
+    parser.add_argument(
+        "--win-length",
+        default=1100,
+        type=int,
+        help="the number of samples between the starts of consecutive frames",
+    )
+    parser.add_argument(
+        "--f-min",
+        default=40.,
+        type=float,
+        help="the number of samples between the starts of consecutive frames",
+    )
+    parser.add_argument(
+        "--min-level-db",
+        default=-100,
+        type=float,
+        help="the min db value for spectrogam normalization",
+    )
+    parser.add_argument(
+        "--n-res-block",
+        default=10,
+        type=int,
+        help="the number of ResBlock in stack",
+    )
+    parser.add_argument(
+        "--n-rnn",
+        default=512,
+        type=int,
+        help="the dimension of RNN layer",
+    )
+    parser.add_argument(
+        "--n-fc",
+        default=512,
+        type=int,
+        help="the dimension of fully connected layer",
+    )
+    parser.add_argument(
+        "--kernel-size",
+        default=5,
+        type=int,
+        help="the number of kernel size in the first Conv1d layer",
+    )
+    parser.add_argument(
+        "--n-freq",
+        default=80,
+        type=int,
+        help="the number of bins in a spectrogram",
+    )
+    parser.add_argument(
+        "--n-hidden",
+        default=128,
+        type=int,
+        help="the number of hidden dimensions",
+    )
+    parser.add_argument(
+        "--n-output",
+        default=128,
+        type=int,
+        help="the number of output dimensions",
+    )
+    parser.add_argument(
+        "--mode",
+        default="waveform",
+        choices=["waveform", "mol"],
+        type=str,
+        help="the type of waveform",
+    )
+    parser.add_argument(
+        "--seq-len-factor",
+        default=5,
+        type=int,
+        help="seq_length = hop_length * seq_len_factor",
+    )
+    parser.add_argument(
+        "--test-samples",
+        default=50,
+        type=float,
+        help="the number of waveforms for testing",
+    )
+    parser.add_argument(
+        "--file-path",
+        default="/private/home/jimchen90/datasets/LJSpeech-1.1/wavs/",
+        type=str,
+        help="the path of audio files",
+    )
+
+    args = parser.parse_args()
+    return args
+
+
+def signal_handler(a, b):
```

Comment:
this is needed to run on cluster, but elsewhere. we won't include this in this pipeline.

Reply:
All this part has been removed.